docs/reference/02-find.md

```diff
@@ -0,0 +1,82 @@
+# Basic API
+
+Find, update and delete records.
+
+The examples use the following prisma schema:
+
+```prisma
+model Post {
+    id        String   @default(cuid()) @id
+    createdAt DateTime @default(now())
+    updatedAt DateTime @updatedAt
+    published Boolean
+    title     String
+    content   String?
+
+    comments Comment[]
+}
+
+model Comment {
+    id        String   @default(cuid()) @id
+    createdAt DateTime @default(now())
+    content   String
+
+    post   Post @relation(fields: [postID], references: [id])
+    postID String
+}
+```
+
+## Reading data
+
+### Find many records
+
+```go
+posts, err := client.Post.FindMany(
+    db.Post.Title.Equals("hi"),
+).Exec(ctx)
+```
+
+If no records are found, this returns an empty array without returning an error (like usual SQL queries).
```

Comment:
"this" is a bit ambiguous. Maybe "FindMany" or "this query" or "the above".

like usual SQL queries => like SELECT queries in SQL.

Reply:
maybe ``err` returns an empty array`